vsintegration/tests/UnitTests/BraceMatchingServiceTests.fs

```diff
@@ -32,8 +32,13 @@ type BraceMatchingServiceTests()  =
         Stamp = None
     }
 
+    // Adding this new-line character at the end of the source seems odd but is required for some unit tests
+    // Todo: fix tests
+    let addNewLine (source: string) =
+        if source.Length = 0 || not (source.[source.Length - 1] = '\n') then source + "\n" else source
+
     member private this.VerifyNoBraceMatch(fileContents: string, marker: string) =
-        let sourceText = SourceText.From(fileContents)
+        let sourceText = SourceText.From(addNewLine fileContents)
```

Comment:
<img width="288" alt="image" src="https://user-images.githubusercontent.com/6309070/49950339-7d035300-feac-11e8-9bd7-21d7975310e1.png">

(I know why this is there, but still... 😢 )

Reply:
We have no choice, but at least we are trying to isolate it in tests. We will work on another PR to rid of this.

Reply:
Yep, I'm aware